frontend/src/components/questions/Question4.tsx

```diff
@@ -1,10 +1,113 @@
-const Question4 = () => {
-    return (
-     <div>
-        <h5>How financially prepared do you feel for retirement?</h5>
-     </div>
-    );
-  };
-  
-  export default Question4;
-  
\ No newline at end of file
+import React from 'react'
+
+import { ToggleButton, ToggleButtonGroup } from '@mui/material'
+import { Trans, useTranslation } from 'react-i18next'
+
+type QuestionProps = {
+  values: { [field: string]: any }
+  setFieldValue: (field: string, value: any, shouldValidate?: boolean) => void
+}
+
+const Question4 = ({ values, setFieldValue }: QuestionProps) => {
+  let { t } = useTranslation('learn')
+  const [value, setValue] = React.useState('')
+
+  const handleChange = (
+    event: React.MouseEvent<HTMLElement>,
+    answerId: string
+  ) => {
+    setValue(answerId)
+    setFieldValue(
+      'retirementTimeframe',
+      values['retirementTimeframe'] === answerId ? '' : answerId ?? ''
+    )
+  }
+
+  return (
+    <div>
+      <h5 className="font-display text-2xl font-light">
+        {t('quiz.questions.question-4.title')}
+      </h5>
+      <ToggleButtonGroup
+        orientation="vertical"
+        exclusive
+        fullWidth={true}
+        className="my-4"
+        value={value}
+        onChange={handleChange}
+        sx={{
+          '& .MuiToggleButton-root:not(:first-of-type)': {
+            borderTop: '1px solid #e1e4e7',
+            borderRadius: '4px',
+          },
+          '.MuiToggleButton-root.Mui-selected': {
+            'backgroundColor': '#004f56',
+            'color': '#f1f1f1',
+            '&.Mui-selected:hover': {
+              backgroundColor: '#004f56',
+            },
+          },
+        }}
+      >
+        <ToggleButton
+          value="CanadaFT"
+          aria-label={t('quiz.questions.question-4.option-1')}
+          className="my-4 font-display text-base font-bold normal-case"
+          selected={values['retirementTimeframe'] === 'CanadaFT'}
+        >
+          {t('quiz.questions.question-4.option-1')}
+        </ToggleButton>
+        <ToggleButton
+          value="CanadaPT6OrMore"
+          aria-label={t('quiz.questions.question-4.option-2-label')}
+          className="my-4 font-display text-base font-bold normal-case"
+          selected={
+            values['retirementTimeframe'] === 'CanadaPT6OrMore'
+          }
+        >
+          <Trans
+            ns="learn"
+            i18nKey="quiz.questions.question-4.option-2"
+            components={{
+              upper: <span className="px-1 underline"></span>,
```

Comment:
According to the canada.ca style guide:

> Use underlining for links only.

https://www.canada.ca/en/treasury-board-secretariat/services/government-communications/canada-content-style-guide.html#wp4-2